workers-site/index.js

```diff
@@ -1,17 +1,16 @@
-import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
+import { getAssetFromKV } from '@cloudflare/kv-asset-handler'
 
-addEventListener("fetch", event => {
-  event.respondWith(handleEvent(event));
-});
+addEventListener('fetch', event => {
+  event.respondWith(handleEvent(event))
+})
 
 async function handleEvent(event) {
   try {
-    return await getAssetFromKV(event);
+    return await getAssetFromKV(event)
   } catch (e) {
-    let pathname = new URL(event.request.url).pathname;
-    return new Response(`"${pathname}" not found`, {
+    let pathname = new URL(event.request.url).pathname
```

Comment:
looks like this pathname is no longer used here?

Reply:
Correct I should remove it thanks!